src/NServiceBus.Core/Features/FeatureActivator.cs

```diff
@@ -36,7 +36,7 @@ public void Add(Feature feature)
             }));
         }
 
-        public FeaturesReport SetupFeatures(IConfigureComponents container, PipelineSettings pipelineSettings)
+        public FeaturesReport SetupFeatures(IConfigureComponents container, PipelineSettings pipelineSettings, RoutingComponent routing)
```

Comment:
what do you consider "special treatment"? Are you relating to this specific code, or the the whole PR?